src/components/Game/Game.tsx

```diff
@@ -166,43 +162,27 @@ export const Game = () => {
 
       <div
         className={
-          gameStatus === "inprogress" ? "block-start" : "block-start-hidden "
+          gameStatus == GameStatus.inprogress
```

Comment:
== ?

Reply:
Yesterday i tried to do fix this - it didn't work. i kept it